src/geometry/shape.rs

```diff
@@ -42,4 +38,22 @@ impl super::Geometry {
             radius,
         }
     }
+
+    pub fn stroke(path: &Path, options:&StrokeOptions) -> Self {
+        let mut buffer = PositionBuilder::new();
+        let builder = &mut BuffersBuilder::new(&mut buffer, stroke_position);
+        let mut tessellator = StrokeTessellator::new();
+        {
+            tessellator.tessellate_path(path, options, builder).unwrap();
```

Comment:
why are we moving this to a separate scope?

Reply:
Cargo cult programming on my part, it was like this in the example.